.github/workflows/canary-e2e.yml

```diff
@@ -0,0 +1,12 @@
+name: Canary CI
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  canary-ci:
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/canary-ci run') }}
+    uses: dai-shi/waku/.github/workflows/e2e.yml@main
```

Comment:
I think `@main` is the default and we can omit it. Oh, it's different syntax. So:

```suggestion
    uses: ./.github/workflows/ci.yml
```

Reply:
IIRC, this should be in a step for visibility. See:

https://github.com/dai-shi/waku/pull/1057/files#diff-742d29bcd0bfa0c762922f6d92f9c0d0818927e44846e67a8575e289560f89b7R10-R11

Reply:
Oh, we need to run both e2e.yml and test.yml. (previously it was combined.)

Reply:
> IIRC, this should be in a step for visibility. See:

this is an invalid github action. I checked it by using the "edit file" button

<img width="916" alt="image" src="https://github.com/user-attachments/assets/b523efa5-6fec-461d-93a4-87ce88d338be" />

So I changed it to not be a step to be valid. 

What is the visibility you're referring to though?

Reply:
Oh, okay. it was about the visibility in the PR page. Let's revisit if we actually see a problem.